exercises/chapter9/test/Main.purs

```diff
@@ -131,7 +131,7 @@ This line should have been automatically deleted by resetSolutions.sh. See Chapt
         Assert.equal (Set.fromFoldable expected) $ Set.fromFoldable actualRelative
       test "one file" do
         let
-          file = Path.concat [ recurseDir, "c/unused.txt" ]
+          file = Path.concat [ recurseDir, "c", Path.sep, "unused.txt" ]
```

Comment:
Does this still work if you omit the `Path.sep` here? I can't test on Windows, but on Mac omitting the separator yields the same result
```
> import Node.Path as Path
> Path.concat [ "c", "unused.txt" ]
"c/unused.txt"

> Path.concat [ "c", Path.sep, "unused.txt" ]
"c/unused.txt"

```

Reply:
Yes, it works. `Path.sep` is redundant.